src/framework/caspool.cpp

```diff
@@ -77,6 +79,38 @@ namespace uima {
   }
 
 
+  CASPool::CASPool(AnnotatorContext *anContext, const AnalysisEngineDescription &taeSpec,
```

Comment:
there's a lot of code duplication between these two constructors. Any manner they can be mixed together?

Reply:
As we discussed I added the new one to replace the old, so I'm waiting for approval to remove/deprecate it

Reply:
Alright, let's do it.